### PR TITLE
 Do not skip Galaxy client build for ``planemo serve``

### LINKED ISSUE #845: Planemo serve doesn't serve up static contents with default command line invocation
`planemo serve <tool_xml>`. Using `--galaxy_branch release_18.05` works fine. Possibly we just need to update ~~the~~ galaxy's master branch ?

### PR DESCRIPTION
**Pull request by nsoranzo**
Fix #845 .

Also, convert an existing `gx_repo` git repository from bare to mirror before doing a `git remote update`.

### CODE REVIEW

File: planemo/galaxy/serve.py [hunk 1]
```diff
@@ -35,6 +37,7 @@ def _serve(ctx, runnables, **kwds):
     if port is None:
         port = network_util.get_free_port()
         kwds["port"] = port
+    kwds['galaxy_skip_client_build'] = False
```
Comment (jmchilton) [inline planemo/galaxy/serve.py]: This should be done in cmd_serve and cmd_shed_serve I think, that way when serve is done here for workflow testing for instance this isn't rebuilt. Does that sound fair?

Comment (nsoranzo) [reply planemo/galaxy/serve.py]: Good point, fixed in a rebase!

**Comment by nsoranzo** [comment]
Finally green!! :tada:

**Comment by bgruening** [comment]
Thanks all!

**Comment by nsoranzo** [comment]
Time for a new Planemo release supporting Galaxy 18.09?

**Comment by jmchilton** [comment]
I let you all down on this - sorry :(. Just released this.

**Pull request merged**